README.adoc

```diff
@@ -7,34 +7,52 @@ image::https://travis-ci.org/hawkular/hawkular-apm.svg?branch=master[Build Statu
 endif::[]
 
 [.lead]
-*Hawkular APM* is the Application Performance Management (APM) module for Hawkular.
+is the *Application Performance Management* solution offering:
 
-The project will provide the capabilities to monitor the flow of invocations
-across servers, tiers, on-premises and in the cloud. It will also enable detailed
+* Distributed Tracing (e.g. for microservices)
+* Application Performance Management
+* Business Transaction Management (currently only using JVM javaagent based approach)
+
+Applications can be instrumented, using a non-intrusive Java agent approach for JVM based applications, or through
+http://zipkin.io/pages/existing_instrumentations.html[Zipkin client libraries] (for polyglot environment).
+
+In other words Hawkular-apm provides the capabilities to monitor the flow of invocations
+across servers, tiers, on-premises and in the cloud. It also enables detailed
 performance analysis to be performed of the individual components that make up an
 application.
 
-== Build
+== Build & Run
 
-To build the project, run:
+To build and run execute:
 
 ```shell
-mvn clean install [ -Ptests ]
+$ mvn clean install -Pdev -DskipTests [-Ptests]
+$ dist/target/hawkular-apm-${version}/bin/standalone.sh [-Djboss.socket.binding.port-offset=1331]
+-> go to http://localhost:[8180|9411]/ and log in with jdoe:password
```

Comment:
vote for 8080, It's their problem. This helps them to configure wildfly with different port